TriblerGUI/widgets/graph_provider.py

```diff
@@ -1,6 +1,8 @@
 """
 Provides a class which encapsulates the graph construction logic.
 """
+from __future__ import division
```

Comment:
Please don't use this. This is very confusing when reading the code if one doesn't know this is being imported.